### ISSUE TITLE
Mouse move not working when using async client

### ISSUE THREAD

**Issue by samiraguiar** [sibson/vncdotool#144]
I'm connecting to a VNC server (qemu) using protocol version 3.8 and vncdotool 0.12.1.

I'm trying to drag the white area as shown bellow (this is the account setup screen on Android Marshmallow)

![before-scroll](https://user-images.githubusercontent.com/13439135/52606702-ad3e8200-2e5a-11e9-88c7-582875e59f4e.png)

Basically I want it to be scrolled up so the hidden contents are visible:
![after-scroll](https://user-images.githubusercontent.com/13439135/52606703-ad3e8200-2e5a-11e9-89a6-900b5007cbfc.png)

However I can't get it to work. My move commands won't work when using vncdotool as a library, but they will work when running from the command line.
Here's my attempt when running as a library:

```python
from vncdotool import api

with api.connect('192.168.255.1::5901') as client:
    client.mouseMove(419, 460)
    client.mouseDown(1)
    client.mouseMove(419, 200)
    client.mouseUp(1)
```

Nothing happens. However, if I run:
```bash
$ vncdo -s 192.168.255.1::5901 mousemove 419 460 mousedown 1 mousemove 419 200 mouseup 1
```

It will scroll correctly as shown in the second image. I know that `mouseDown` and `mouseUp` calls work because if I call `client.mouseDown(1)`, start a vncviewer session and move my mouse it will drag that area, meaning the left mouse button is being held down.

I tried to add sleep calls between each of the calls and before exiting the `with` block to avoid concurrency issues or to avoid exiting before all commands were sent, but it was to no avail.

My best guess is that the threaded proxy might be interfering as `command.py` doesn't seem to use it. However I'm basically clueless about what can be wrong.
What could be causing this issue?

**Comment by samiraguiar** [comment]
I've done a few more tests with the mouse drag function and this works:
```bash
$ vncdo -s 192.168.255.1::5901 drag 419 460 mousedown 1 mousemove 419 200 mouseup 1
```

But this doesn't:
```bash
$ vncdo -s 192.168.255.1::5901 drag 419 460 mousedown 1 drag 419 200 mouseup 1
```

**Comment by samiraguiar** [comment]
I believe I have found the problem. `api.py` doesn't add a delay between calls to the client methods, while `command.py` [does](https://github.com/sibson/vncdotool/blob/d308133ee66a95c849f2d9b05faeeb87fc120561/vncdotool/command.py#L208). Adding `time.sleep()` calls like I did doesn't work maybe because commands are run in a separate thread.

Adding a second call to `callFromThread` in [`proxy_call`](https://github.com/sibson/vncdotool/blob/d308133ee66a95c849f2d9b05faeeb87fc120561/vncdotool/api.py#L100) seems to solve the problem:
```python
def proxy_call(*args, **kwargs):
    reactor.callFromThread(self.factory.deferred.addCallbacks,
                           callback, errback, args, kwargs)
    reactor.callFromThread(self.factory.deferred.addCallback,
                           self.factory.protocol.pause, 0.01)
    try:
        result = self.queue.get(timeout=self._timeout)
    ...
```

I'm not sure if this lack of a pause in `api.py` is by design, though.

**Comment by pevogam** [comment]
@samiraguiar Do you think there is a fix we can propose other than duplicating the call above?

**Comment by samiraguiar** [comment]
After #163 was merged, I managed to solve the aforementioned problem by using the drag and drop function which is now working correctly.
Although it is true that the command line tool will pause between each command while the exposed API will not, I don't know to what extend this is by design, therefore I'm closing this issue. But feel free to reopen it if needed, @sibson.
